plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/collection-chooser.tsx

```diff
@@ -54,64 +65,162 @@ export const applyCollection = (
 };
 
 const CollectionButton = ( {
-	active = false,
 	title,
 	icon,
 	description,
 	onClick,
 }: CollectionButtonProps ) => {
-	const variant = active ? 'primary' : 'secondary';
-
 	return (
-		<Button
-			className="wc-blocks-product-collection__collection-button"
-			variant={ variant }
-			onClick={ onClick }
-		>
-			<div className="wc-blocks-product-collection__collection-button-icon">
-				{ icon }
-			</div>
-			<div className="wc-blocks-product-collection__collection-button-text">
+		<Tooltip text={ description } placement="top">
+			<Button
+				className="wc-blocks-product-collection__collection-button"
+				onClick={ onClick }
+			>
+				<div className="wc-blocks-product-collection__collection-button-icon">
+					<Icon icon={ icon } />
+				</div>
 				<p className="wc-blocks-product-collection__collection-button-title">
 					{ title }
 				</p>
-				<p className="wc-blocks-product-collection__collection-button-description">
-					{ description }
-				</p>
+			</Button>
+		</Tooltip>
+	);
+};
+
+const CreateCollectionButton = ( props: CollectionButtonProps ) => {
+	const { description, onClick } = props;
+
+	return (
+		<div className="wc-blocks-product-collection__collections-create">
+			<span>{ __( 'or', 'woocommerce' ) }</span>
+			<Tooltip text={ description } placement="top">
+				<Button onClick={ () => onClick() }>
```

Comment:
It can be simplified:

```suggestion
				<Button onClick={onClick}>
```